source/trackers.json

```diff
@@ -859,6 +865,17 @@
         "samsungvisioncloud.com": "samsungapps",
         "sbixby.com": "samsungapps",
         "sectigo.com": "sectigo",
+        "myshopify.com": "shopify",
+        "shop.app": "shopify",
+        "shopifyapps.com": "shopify",
+        "shopifycdn.net": "shopify",
+        "shopify.com.au": "shopify",
+        "shopify.com.mx": "shopify",
+        "shopify.co.za": "shopify",
+        "shopify.dev": "shopify",
+        "shopifynetwork.com": "shopify",
+        "shopifypreview.com": "shopify",
+        "shopifysvc.com": "shopify",
```

Comment:
`"shopifysvc.com"` looks like tracker or analytics. maybe we can rewrite `"shopify_stats"` with category 6 - site_analytics (+ `stats.shopify.com`, `v.shopify.com`)

Reply:
> `"shopifysvc.com"` looks like tracker or analytics. maybe we can rewrite `"shopify_stats"` with category 6 - site_analytics (+ `stats.shopify.com`, `v.shopify.com`)